src/monster.cpp

```diff
@@ -458,12 +456,88 @@ int monster::print_info(WINDOW* w, int vStart, int vLines, int column) const
 
 std::string monster::extended_description() const
 {
-    // @todo Add tons of info here
-    std::string ret = string_format( _ ( "This is a %s" ), name().c_str() );
-    ret += "\n--\n";
+    std::ostringstream ss;
+
+    nc_color color = c_white;
```

Comment:
Indeed, I overlooked that. Nevertheless, it outlights the drawback of C-ish approach: reduced readability. The function shouldn't be accepting any arguments, instead it should be returning a `pair` of values (like, for example, `hp_description()` does). Of course, the code is pre-existed, so that's just a side note.